### PR TITLE
adding documentation updates for missing methods

### PR DESCRIPTION
**Pull request by egreer**
Adds some documentation for phi_allowed, log configuration, and the include / exclude behavior

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,18 @@ Or install it yourself as:
...
 
+## Initialize
+
+Create an initializer to configure the PHI log file location. Example:
+
+ `config/initializers/phi_attrs.rb`
+
+```ruby
+PhiAttrs.configure do |conf|
+  conf.log_path = Rails.root.join("log", "phi_access_#{Rails.env}.log")
```
Comment (Crisfole) [inline README.md]: :+1: are there other configurable features?

File: README.md [hunk 2]
```diff
@@ -66,13 +78,44 @@ PatientInfo.allow_phi!("allowed_user@example.com", "Customer Service")
 
 As of version `0.1.5`, a block syntax is available. As above, this is available on both class and instance levels.
 
-Note the lack of a `!` at the end—these methods don't necessarily get along well with the mutating (bang) methods!
+Note the lack of a `!` at the end. These methods don't necessarily get along well with the mutating (bang) methods!
```
Comment (Crisfole) [inline README.md]:
```suggestion
Note the lack of a `!` at the end. These methods should not be used alongside the mutating (bang) methods! We recommend using the block syntax whenever possible.
```

File: README.md [hunk 3]
```diff
@@ -66,13 +78,44 @@ PatientInfo.allow_phi!("allowed_user@example.com", "Customer Service")
...
+
+  include_in_phi :birthday, :note
+
+  def birthday
+    Time.current
+  end
+
+  attr_accessor :note
+end
+```
```
Comment (Crisfole) [inline README.md]: I recommend you add an example of before/after here. Something like:

```
class PatientInfo < ActiveRecord::Base
  phi_model
end

pi = PatientInfo.new(first_name: "Ash", last_name: "Ketchum")
pi.created_at
# PHIAccessException!
pi.last_name
# PHIAccessException!
pi.allow_phi "Ash", "Testing PHI Attrs" { pi.last_name }
# "Ketchum"

class PatientInfoTwo < PatientInfo
  exclude_from_phi :created_at
end

pi = PatientInfo.new(first_name: "Ash", last_name: "Ketchum")
pi.created_at
# current time
pi.last_name
# PHIAccessException!
pi.allow_phi "Ash", "Testing PHI Attrs" { pi.last_name }
# "Ketchum"

class PatientInfoThree < PatientInfoTwo
  include_in_phi :created_at #Changed our mind
end

pi = PatientInfo.new(first_name: "Ash", last_name: "Ketchum")
pi.created_at
# PHIAccessException!
pi.last_name
# PHIAccessException!
pi.allow_phi "Ash", "Testing PHI Attrs" { pi.last_name }
# "Ketchum"
```

File: README.md [hunk 4]
```diff
@@ -98,6 +141,53 @@ patient.patient_info.first_name
...
+This also works if access was granted at the class level
+
+```ruby
+patient = Patient.new
+patient.phi_allowed? # => false
+Patient.allow_phi!('user@example.com', 'reason')
+patient.phi_allowed? # => true
+```
+
+There is currently no class level equivalent for `phi_allowed?`.
```
Comment (Crisfole) [inline README.md]: This would be stupidly easy to implement.

Comment (egreer) [reply README.md]: Yes there is a task for it.

**Pull request merged**